### DISCUSSION TITLE
Module '"@react-aria/utils"' has no exported member 'RouterProvider'.

### DISCUSSION THREAD

**Discussion by mgo-af** [adobe/react-spectrum#5197] [Q&A] [answered]
Hi team 👋 

## Problem

We're consumers of `react-aria` and `react-aria-components` and we have a simple consumer app test that checks for types augmentations in CI. This error started to get thrown recently and I was wondering if anyone else had encountered it:

```
Error: node_modules/react-aria-components/node_modules/react-aria/dist/types.d.ts(46,35): error TS2305: Module '"@react-aria/utils"' has no exported member 'RouterProvider'.
```

## What have we tried

Updating to RAC `beta`, emptying caches.

**Comment by reidbarber** [comment, marked as answer]
This is being investigated over in https://github.com/adobe/react-spectrum/issues/5194
